### PR TITLE
Html 5 form

### PR DESCRIPTION
**Pull request by IrynaY**
### HTML5: Forms

[Demo](https://irynay.github.io/frontend-2019-homeworks/) |
[Code base](https://github.com/IrynaY/frontend-2019-homeworks/tree/html-5-form/submissions/IrynaY/html5-form/app)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/IrynaY/html5-form/app/index.html [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+                                name="card-number"
+                                pattern="\d*"
+                                minlength="16"
+                                maxlength="16"
+                                placeholder="4242 4242 4242 4242"
+                                required>
+                    </div>
+                    <div class="card-info">
+                        <label for="expiry-date" class="title">Expiry date</label>
+                        <input type="text"
```
[the remark this answers is not in the document]
Comment (IrynaY) [reply submissions/IrynaY/html5-form/app/index.html]: Thanks for review. I have question about input["data"].
If a set type as "date", input has own calendar. Shod i add a regexp in this case?

**Comment by IrynaY** [comment]
> well done on the desktop version!
> please take a look at the comments.
> 
> p.s. how are things going with the mobile version?

Using tables was a bad idea for mobile 😅

File: submissions/IrynaY/html5-form/app/index.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (yevhenorlov) [reply submissions/IrynaY/html5-form/app/index.html]: @IrynaY regexp isn't necessary, `required` attribute should be enough

[the remark this answers is not in the document]
Comment (IrynaY) [reply submissions/IrynaY/html5-form/app/index.html]: @yevhenorlov thanks)

**Comment by IrynaY** [comment]
I am done. Please, review)

**Pull request merged**